autosklearn/evaluation/train_evaluator.py

```diff
@@ -856,13 +796,9 @@ def _partial_fit_and_predict_iterative(
                 if add_model_to_self:
                     self.model = model
 
-                train_loss = self._loss(
-                    self.Y_train.iloc[train_indices]
-                    if hasattr(self.Y_train, "iloc")
-                    else self.Y_train[train_indices],
-                    Y_train_pred,
-                )
-                loss = self._loss(self.Y_train[test_indices], Y_optimization_pred)
+                train_loss = self._loss(y, Y_train_pred, X_data=X)
+                loss = self._loss(y_test, Y_optimization_pred, X_data=X)
```

Comment:
Should `X_data` here be `X_test` instead of `X`?

Reply:
Yup, that seems like it should, nice spot!